docs/MultiTableDatabase-batchsource.md

```diff
@@ -39,6 +39,8 @@ Pattern syntax is specific to the type of database that is being connected to.
 **tableNameField**: The name of the field that holds the table name.
 Must not be the name of any table column that will be read. Defaults to 'tablename'.
 
+**whereClause** The where clause: i.e. ```where updated_at > '2018-08-20 00:00:00'```
```

Comment:
Though it's probably obvious, think it's worth mentioning that this will be applied to every table, meaning every table needs to have the column(s) mentioned in the where clause.